src/Mod/Sketcher/App/SketchObjectPyImp.cpp

```diff
@@ -2530,9 +2493,7 @@ int SketchObjectPy::setCustomAttributes(const char* attr, PyObject* obj)
         if (strcmp(attr, "Geometry") == 0) {
             getSketchObjectPtr()->rebuildVertexIndex();
         }
-
         return 1;
     }
-
     return 0;

```

Comment:
There's a lot of these white space removals in this PR.
While it's technically not wrong I think it would be good to avoid these type of changes when cleaning code in the future. Otherwise you will remove them and someone else will add them back again. It will be a never ending back and forth.

That said, I don't think you need to go back and change it in this PR.